components/molecules/m-update-personal-data.vue

```diff
@@ -0,0 +1,171 @@
+      <SfButton class="form__button" @click.native="updatePersonalData">
+        {{ $t('Update personal data') }}
+      </SfButton>
+    </div>
+    <p class="notice">
+      {{ $t('At Brand name, we attach great importance to privacy issues and are committed to protecting the personal data of our users. Learn more about how we care and use your personal data in the') }}
+      <a href="">{{ $t('Privacy Policy') }}</a>.
+    </p>
+  </div>
+</template>
+
+<script>
+import { SfInput, SfButton } from '@storefront-ui/vue';
+import { required, minLength, email } from 'vuelidate/lib/validators';
+import { unicodeAlpha } from '@vue-storefront/core/helpers/validators';
+
+export default {
+  name: 'MUpdatePersonalData',
+  components: {
+    SfInput,
+    SfButton
+  },
+  data () {
+    return {
+      firstName: '',
+      lastName: '',
+      email: ''
+    }
+  },
+  methods: {
+    updatePersonalData () {
+      this.$v.$touch();
+      if (this.$v.$invalid) {
+        this.$store.dispatch('notification/spawnNotification', {
+          type: 'error',
+          message: this.$t('Please fix the validation errors'),
+          action1: { label: this.$t('OK') }
+        });
+        return;
+      }
+      let updatedProfile = JSON.parse(JSON.stringify(this.$store.state.user.current))
+      updatedProfile.firstname = this.firstName
+      updatedProfile.lastname = this.lastName
+      updatedProfile.email = this.email
+      this.$bus.$emit('myAccount-before-updateUser', updatedProfile)
+    }
+  },
+  beforeMount () {
+    let currentUser = this.$store.state.user.current;
+    this.firstName = currentUser.firstname
+    this.lastName = currentUser.lastname
+    this.email = currentUser.email
+  },
+  validations: {
+    firstName: {
+      required,
+      minLength: minLength(2),
+      unicodeAlpha
+    },
+    lastName: {
+      required,
+      unicodeAlpha
+    },
+    email: {
+      required,
+      email
+    }
+  }
+};
+</script>
+
+<style lang="scss" scoped>
+@import "~@storefront-ui/vue/styles";
+@mixin for-desktop {
+  @media screen and (min-width: $desktop-min) {
+    @content;
+  }
+}
+.form {
+  @include for-desktop {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+  }
+  &__element {
+    margin-bottom: $spacer-extra-big;
+    @include for-desktop {
+      flex: 0 0 100%;
+    }
+    &--half {
+      @include for-desktop {
+        flex: 1 1 50%;
+      }
+      &-even {
+        @include for-desktop {
+          padding-left: $spacer-extra-big;
+        }
+      }
+    }
+  }
+  &__button {
+    width: 100%;
+    @include for-desktop {
+      width: auto;
+    }
+  }
+}
+.message,
+.notice {
+  font-family: $body-font-family-primary;
+  font-weight: $body-font-weight-primary;
+  line-height: 1.6;
+}
+.message {
+  margin: 0 0 $spacer-extra-big 0;
+  font-size: $font-size-regular-mobile;
+  @include for-desktop {
+    font-size: $font-size-regular-desktop;
+  }
+  &__label {
+    font-weight: 400;
+  }
```

Comment:
`.message__label` class is not used in template

Reply:
Fixed